examples/contract/simple_auction.js

```diff
@@ -133,7 +133,7 @@ define('contract/simple_auction', ['Q'], function(Q) {
                    moneyDstP: purse,
                    moneyNeeded: maxBid,
                    good: good,
-                   cancellationP: ...
+                   cancellationP: {},
```

Comment:
add a `TODO ...` comment here, I think the example isn't meant to be functional yet, and we should retain a reminder of that fact.